app/controllers/sms_controller.rb

```diff
@@ -75,9 +75,14 @@ def deliver_reply(reply)
       if incoming_adapter.reply_style == :via_adapter
         outgoing_adapter.deliver(reply)
       else # reply via response
-        incoming_adapter.prepare_message_for_delivery(reply)
+        begin
+          raise ENV["STUB_REPLY_ERROR"].constantize if Rails.env.test? && ENV["STUB_REPLY_ERROR"].present?
+          incoming_adapter.prepare_message_for_delivery(reply)
+        rescue StandardError
+          reply.error_message = ENV["STUB_REPLY_ERROR"]
```

Comment:
`ENV["STUB_REPLY_ERROR"]` is just a testing tool. So it doesn't really make sense to be setting it as the error message. This code will be running even in production. I encourage you to think through what is really going on here. If you don't understand let me know.